services/github-actions.js

```diff
@@ -1,7 +1,35 @@
 "use strict";
 const axios = require("axios");
+const createError = require("http-errors");
+
+const utils = require("../utils");
+
+function populateWorkflowWithEnvPat(workflow) {
+  try {
+
+    const config = utils.getConfig()
+    const hasEnvPat = config.hasEnvPat
+
+    if (!hasEnvPat) {
+      return workflow
+    }
+    const wfConfigPat = config.pats[workflow.name]
+
+    if (wfConfigPat) {
```

Comment:
Could you log some warning if hasEnvPat is set to true but `config.pats[workflow.name]` doesn't exist. That way we can warn about typos in the config file.